packages/react-router/src/components/Router/Router.tsx

```diff
@@ -21,7 +21,7 @@ export default function Router({location, children}: Props) {
   }
 
   const locationString =
-    typeof location === 'object' ? location.href : location;
+    typeof location === 'object' ? location.pathname : location;
```

Comment:
whyyy, like whyyyy

Reply:
In react router's doc. You are suppose to use [`req.url`](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/StaticRouter.md#location-string)

The example for the url being `In the request www.example.com/index.html?a=1&b=2, req.url will contain /index.html?a=1&b=2.` which does not explain why pathname works